fakeredis/_server.py

```diff
@@ -2129,6 +2129,20 @@ def zincrby(self, key, increment, member):
     def zlexcount(self, key, min, max):
         return key.value.zlexcount(min.value, min.exclusive, max.value, max.exclusive)
 
+    @command((Key(ZSet),), (Int,))
+    def zpopmin(self, key, count=None):
+        zset = key.value
+        min_score = ScoreTest.decode(b'-inf').lower_bound
+        max_score = ScoreTest.decode(b'inf').upper_bound
+        items = list(zset.irange_score(min_score, max_score))
```

Comment:
I don't think it's necessary to construct and pass `min_score` and `max_score`: you should be able to just pass `None` for both.